xcvm/lib/core/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+authors = ["Composable Developers"]
+edition = "2021"
+homepage = "https://composable.finance"
+name = "xcvm-core"
+version = "0.1.0"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false }
+scale-info = { version = "2.1.1", default-features = false, features = [
+  "derive",
+] }
+schemars = { version = "0.8.1", optional = true }
+serde = { version = "1", default-features = false, features = ["derive"] }
+serde_json = { version = "1", default-features = false, features = ["alloc"] }
```

Comment:
does serde need the alloc feature as well? or does serde_json:alloc bring in serde:alloc automatically

Reply:
not sure about that, but isn't serde just here for the `Serialize` and `Deserialize` traits + derive?

Reply:
I was just referencing this: https://serde.rs/no-std.html, but it seems that serde_json/alloc brings in serde/alloc (https://github.com/serde-rs/json/blob/master/Cargo.toml#L54)

Reply:
Remove serde_json and added alllloc